src/hooks/useSubscriptionPlan.ts

```diff
@@ -0,0 +1,30 @@
+import {useMemo} from 'react';
+import {useOnyx} from 'react-native-onyx';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
+
+function useSubscriptionPlan() {
+    const [policies] = useOnyx(ONYXKEYS.COLLECTION.POLICY);
+
+    // Filter workspaces in which user is the admin and the type is either corporate (control) or team (collect)
+    const adminPolicies = useMemo(
+        () =>
+            Object.values(policies ?? {}).filter(
+                (policy) => policy?.role === CONST.POLICY.ROLE.ADMIN && (CONST.POLICY.TYPE.CORPORATE === policy?.type || CONST.POLICY.TYPE.TEAM === policy?.type),
```

Comment:
Maybe we can use existing utility methods to make it more readable? For example [isPolicyAdmin](https://github.com/Expensify/App/blob/ca2d4b5b3b02b691c6ec8ebe2ecc6041e4ec2ea8/src/libs/PolicyUtils.ts#L138) and [isPaidGroupPolicy](https://github.com/Expensify/App/blob/ca2d4b5b3b02b691c6ec8ebe2ecc6041e4ec2ea8/src/libs/PolicyUtils.ts#L276).

Reply:
Great idea, thanks!